src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+@import 'variables';
+
+.card {
+  width: $card-width;
+  height: $card-height;
+  border: $card-border;
+  border-radius: $card-border-radius;
+  box-shadow: $card-shadow;
+  font-family: $font-family;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: $card-padding;
+  background-color: #fff;
+  transition: box-shadow 0.3s;
+
+  &:hover {
+    box-shadow: 0 8px 16px rgba(0, 0, 0, 0.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__title {
+    display: block;
+    width: 166px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+    margin-bottom: 4px;
+    color: $card-title-color;
+  }
+
+  &__code {
+    display: block;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $card-code-color;
+    margin-bottom: 16px;
+    align-self: flex-start;
+    margin-left: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    width: 166px;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
Add default `font-family` and `font-size` to your `html`, not to repeat it in every class